src/database/queries/posts/searchPostQuery.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-undef */
+const connection = require('../../config/connection');
+
+const searchPostQuery = (searchTitle) => {
+  sql = {
```

Comment:
Use `const` or `let` to declare the `sql` variable inside the searchPostQuery function to avoid any potential leakage into the global scope.